update wizard with new changes and ims=gs (#9668)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/wizard-home.md

<WizardFeedbackCallout />

<DocCarousel slidesPerView={1}>
<Lightbox src="/img/docs/dbt-platform/wizard-home-empty.png" width="95%" title="dbt Wizard home tab — empty state with quick-start prompts" />

> <Lightbox src="/img/docs/dbt-platform/wizard-home-agent.png" width="95%" title="dbt Wizard home tab — agent refactoring a model with an inline SQL diff" />

Reviewer: Hiya Mirna

The title says “with an inline SQL diff,” but the updated screenshot shows the agent chat, a lineage graph, and a modified-files list, not an inline SQL diff. Maybe retitle to match what’s shown, or swap the image for one that shows the SQL diff.

> <Lightbox src="/img/docs/dbt-platform/wizard-home-agent.png" width="95%" title="dbt Wizard home tab — agent refactoring a model with an inline SQL diff" />

Author: ```suggestion
<Lightbox src="/img/docs/dbt-platform/wizard-home-agent.png" width="95%" title="dbt Wizard agent refactoring a docs github model for tech writers :) " />
```
